fig/tendermint/codec/v1/codec.proto

```diff
@@ -49,7 +49,7 @@ message TransactionData {
 }
 
 message Block {
-  Header       header      = 1 [(gogoproto.nullable) = false];
+  Header header = 1 [(gogoproto.nullable) = false];
```

Comment:
formatting

Reply:
The current version is actually a result of running the formatter :) I think the `reserved` keyword breaks the alignment.